### ISSUE TITLE
ds_meta function stopped working

### ISSUE THREAD

**Issue by jenicholson** [domoinc/rdomo#10]
I frequently use the ds_meta function to get the meta data before pulling the data.  This function has stopped working.  I can still pull the data with the ds_get or ds_query functions but when I use the ds_meta function as below I always get a list with the elements $status=403, $statusReason="Forbidden", $message="Access is denied", $toe="V9EF32FE3V-20NDM-JTL2I".  Can you please help?

```
library(rdomo) 
#client ID and secret stored in .Renviron file. Do not share. 
domo <- rdomo::Domo() 
pipeline_id1 <- '<insert valid dataset id>' 
data1_info <- domo$ds_meta(pipeline_id1)
``
![rdomo_access_denied](https://user-images.githubusercontent.com/10472195/178533427-3d9eacea-4737-4b0e-82d2-55774ab394f4.png)
`

**Comment by jeremydmorris** [comment]
That is odd, has your user role changed recently? What permissions are allowed on your API info? I think with that information I can ask to see what might have happened.

**Comment by jenicholson** [comment]
No, my permissions are for a "Participant+" role.  We tried changing my permissions temporarily to full "admin" and the get_meta function worked correctly.  I'm not sure what changed with respect to the "Participant+" role but now we know it is a permissions issue.  I will work with our System Administrators to figure out a solution.
